content/blueprints/spec-policies.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: bt_wiki
+title: Policies
+category: Blueprints
+draft: false
+weight: 1200
+
+---
+
+`policies` provide a way of configuring reusable behavior by referencing [groups]({{< relref "blueprints/spec-groups.md" >}}) for which a policy applies.
+
+{{% gsNote title="Note" %}}
+The top level `policies` section described in this page, should not be confused with policies defined in the top level `policy_types` and the `policies`
+section under groups.
+
+The confusing similar names is something that we are aware of and will work to consolidate in the future. For now, try to keep the differences in mind.
+{{% /gsNote %}}
+
+{{% gsNote title="Note" %}}
+At the moment, the only supported policy type is the built-in `cloudify.policies.scalable`.
+{{% /gsNote %}}
+
+# Declaration
+
+{{< gsHighlight  yaml >}}
+policies:
+  my_scaling_policy:
+    type: cloudify.policies.scalable
+    properties:
+      ...
+    targets: ...
+{{< /gsHighlight >}}
+
+
+# Policy Schema
+
+Keyname     | Required | Type        | Description
+----------- | -------- | ----        | -----------
+type        | yes      | string      | The policty type. (At the moment, only `cloudify.policies.scalable` is supported)
```

Comment:
policty -> policy

also, I think we should remove everything in parentheses as it  is already mentioned in a note above. Once this is not the case and there are additional policy types, it will be better to only remove the note.